src/lib/components/Cytoscape.react.js

```diff
@@ -676,7 +778,38 @@ Cytoscape.propTypes = {
          */
         boundingBox: PropTypes.object,
     }),
-
+    /**
+     * Define a custom context menu
+     */
+    contextMenu: PropTypes.arrayOf(
+        PropTypes.shape({
+            id: PropTypes.string.isRequired,
+            content: PropTypes.string.isRequired,
+            tooltipText: PropTypes.string,
+            coreAsWell: PropTypes.string,
+            selector: PropTypes.string,
+            onClick: PropTypes.oneOfType([
```

Comment:
Now that these two properties have different names than the property referring to the function itself, you can remove the function type